test/e2e/crcsuite/crcsuite.go

```diff
@@ -114,10 +116,10 @@ func FeatureContext(s *godog.Suite) {
 
 		if bundleLocation == "" {
 			fmt.Println("Expecting the bundle to be embedded in the CRC executable.")
```

Comment:
We should change this printout too (or remove it, although it's useful if stuff goes wrong to know what was "expected" without reading any code).